spec/services/by_month_project_fetcher_service_spec.rb

```diff
@@ -32,14 +32,32 @@
     end
   end
 
+  describe "#revised" do
+    it "returns only the projects where the date was the given month but has since changed" do
+      mock_all_academies_api_responses
+
+      project = create(:conversion_project, conversion_date: Date.parse("2025-6-1"), conversion_date_provisional: false)
+      create(:date_history, project: project, previous_date: Date.parse("2025-1-1"), revised_date: Date.parse("2025-6-1"))
+      create(:date_history, project: project, previous_date: Date.parse("2025-1-1"), revised_date: Date.parse("2025-1-1"))
+
+      other_project = create(:conversion_project, conversion_date: Date.parse("2025-6-1"), conversion_date_provisional: false)
+      create(:date_history, project: project, previous_date: Date.parse("2025-1-1"), revised_date: Date.parse("2025-6-1"))
+
+      projects = described_class.new.revised(1, 2025)
+
+      expect(projects).to include project
+      expect(projects).not_to include other_project
+    end
+  end
+
   describe "#confirmed_openers_by_team" do
     context "with pre fetching disabled for this test" do
       before do
         mock_all_academies_api_responses
       end
 
       let(:user_1) { create(:user, :caseworker, team: "regional_casework_services") }
-      let(:user_2) { create(:user, :caseworker, team: "london") }
+      let(:user_2) { create(:user, :regional_delivery_officer, team: "london") }
```

Comment:
I feel like this change doesn't belong in this commit, it's to do with user factories and not scopes. Did it creep in accidentally?